packages/components/src/templates/next/components/complex/Infopic/Infopic.tsx

```diff
@@ -9,14 +9,14 @@ import { ImageClient } from "../Image"
 const infopicStyles = tv({
   slots: {
     container:
-      "grid min-h-[360px] bg-base-canvas-backdrop [grid-template-areas:'img''content'] [grid-template-rows:auto_1fr] lg:grid-cols-2",
+      "grid min-h-[360px] bg-base-canvas-backdrop [grid-template-areas:'img''content'] [grid-template-rows:auto_1fr] lg:grid-cols-2 lg:[grid-template-rows:auto]",
     image: "inset-0 h-full w-full object-cover lg:absolute",
     imageContainer:
-      "relative max-h-[400px] w-full [grid-area:img] lg:max-h-full",
+      "relative max-h-[400px] min-h-[200px] w-full [grid-area:img] lg:max-h-full",
```

Comment:
Hmm, might be good enough to have height 100% (only in lg viewport) instead of setting a min height. This way, the container will fill up the height (which means the image will fill up the height)

Reply:
oh but then in small screens, won't images that are horizontally long have a super small height?

Reply:
Can have both maybe?
But might be better to just prevent admins from uploading those sort of pictures haha